KenticoCloud.Delivery/KenticoCloud.Delivery.csproj

```diff
@@ -35,6 +35,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="1.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Polly" Version="6.0.1" />
```

Comment:
I'm getting the following errors when opening the project.

Could you please find the reason why and try to fix it? Maybe it'd help to just switch to netstandard2.0 or updating some references. I'd be happy with either.


![image](https://user-images.githubusercontent.com/9810625/41986794-7c9f3806-7a37-11e8-8897-ae70ef8a9b6f.png)

Reply:
Sure, will do.